test/Winton.Extensions.Configuration.Consul.Test/Extensions/KVPairQueryResultExtensionsTests.cs

```diff
@@ -201,8 +207,9 @@ private void ShouldNotParseIfConfigBytesIsNull()
                 _parser
                     .Setup(p => p.Parse(It.IsAny<Stream>()))
                     .Returns(new Dictionary<string, string>());
+                _source.KeyToRemove = "path/test";
 
-                result.ToConfigDictionary("path/test", _parser.Object);
+                var config = result.ToConfigDictionary(_source.ConvertToConfig);
```

Comment:
We can also do the same here and in this case it's an even bigger win because we can write a proper expectation and not just a mock verify. For example:

```csharp
var config = result.ToConfigDictionary(_ => new Dictionary { {"key", "value"} });

config.Should().BeEmpty();
```